### ISSUE TITLE
support for changing resolutions + multiple instances?

### ISSUE THREAD

**Issue by not-nullptr** [rozniak/xfce-winxp-tc#226]
hello! as of right now, wintc-taskband does not support changing resolutions or multiple instances, which is problematic for running in vnc. is this planned?

**Comment by rozniak** [comment]
There is #220  regarding the taskband not repositioning following a display layout change - I don't know about multiple instances though, you mean multiple under the same user? :thinking:

**Comment by not-nullptr** [comment]
@rozniak yup. would be very helpful for running this over vnc, as of right now it just doesn't open

**Comment by rozniak** [comment]
@not-nullptr I'm not super familiar with VNC - does it create a session of its own? Like if you are logged in at the terminal with one session, and logged in remotely in another session - both under the same user?

If you can give me some info on how to reproduce, like what VNC server in use, some configs to replicate the issue, I'll see if I can have a look.


Tagging both taskband and run applications, as this will be a general bug with my usage of `GtkApplication`/`GApplication` (`run` is also intended to be single instance and will be affected by this problem I expect)
